src/app/portlets/content-types/fields/content-type-fields-variables/content-type-fields-variables.component.scss

```diff
@@ -0,0 +1,21 @@
+::ng-deep {
+    .ui-datatable {
+        .ui-datatable-thead tr {
+            th {
+                padding: .5em;
+                text-align: left;
+    
+                &:last-child {
+                    width: 108px;
```

Comment:
Why this set width? set widths mean "not responsive" we have to be careful about it.